plaidml/op/lib/ops.cc

```diff
@@ -1642,7 +1642,34 @@ Value explicit_padding(const Value& value) {
       auto padval = args[4].as_tensor();
       O = Contraction(O_dims, O_idxs).assign(I(I_idxs)).init(padval);
     } break;
-    case PadMode::EDGE:
+    case PadMode::EDGE: {
+      IVLOG(2, "Edge padding requested");
+      O = I;
+
+      std::vector<TensorIndex> Lo_idxs(I.rank());
+      std::vector<TensorIndex> Hi_idxs(I.rank());
+      std::vector<TensorDim> Lo_dims(I.rank());
+      std::vector<TensorDim> Hi_dims(I.rank());
+
+      for (size_t i = 0; i < I.rank(); ++i) {
+        std::vector<TensorDim> O_dims(I.rank());
+        O.bind_dims(O_dims);
+
+        std::vector<TensorIndex> lo_idxs = Lo_idxs;
```

Comment:
I find having both `lo_idxs` and `Lo_idxs` as variable names confusing. Perhaps rename as `src_lo_idxs` and `dst_lo_idxs` respectively?